source/slang/vm.cpp

```diff
@@ -846,6 +846,7 @@ void resumeThread(
             }
             break;
 
+#if 0
```

Comment:
#if 0 because it might come back/be useful?

Reply:
No, that was an oversight where I `#ifdef`d out the code during development and then never went back and deleted it. Honestly that whole file needs to be deleted at this point.